keyboards/lazydesigners/the50/the50.h

```diff
@@ -0,0 +1,35 @@
+#ifndef THE50_H
+#define THE50_H
+
+#include "quantum.h"
+
+#define ____ KC_NO
```

Comment:
Please change this to a sequence of X's, or use KC_NO directly.

```suggestion
#define XXXX KC_NO
```

or 

```suggestion
```

then edit the rest of the file accordingly.